.travis.yml

```diff
@@ -1,9 +1,16 @@
 # Configuration script to Travis-CI
+arch:
+     - amd64
+     - ppc64le
 language: python
 python:
   - "3.6"
   - "3.7"
-  - "3.8"
+  - "3.8"  
+before_install:
+  - curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py > get-poetry.py
+  - python get-poetry.py -y --version 1.0.0b3
+  - source ~/.poetry/env  
```

Comment:
Hm, I wouldn't like to depend on poetry. Is it needed to support additional architectures?